src/category.rs

```diff
@@ -1,3 +1,8 @@
+//! The category module handles retrieving Categories. A Category is a ruleset for a Game and may
+//! contain Runs.
+//!
+//! https://github.com/glacials/splits-io/blob/master/docs/api.md#category
```

Comment:
You can use markdown for these links, so maybe it should just be `[API Documentation](https://github.com/glacials/splits-io/blob/master/docs/api.md#category)`